media-sound/pulseaudio/pulseaudio-14.99.1_p20210607.ebuild

```diff
@@ -0,0 +1,351 @@
+
+# NOTE: dev-libs/libpcre header will be used if found but no linking is done on non-Windows
+BDEPEND="
+	doc? ( app-doc/doxygen )
+	orc? ( >=dev-lang/orc-0.4.15 )
+	system-wide? ( dev-util/unifdef )
+	test? ( >=dev-libs/check-0.9.10 )
+	dev-libs/libpcre
+	sys-devel/gettext
+	sys-devel/m4
+	virtual/libiconv
+	virtual/libintl
+	virtual/pkgconfig
+"
+
+DOCS=( NEWS README ) # todo is useless to install
+
+pkg_pretend() {
+	if [[ -n ${COMMIT} && -z ${SNAPSHOT_PV} || -z ${COMMIT} && -n ${SNAPSHOT_PV} ]]; then
+		eerror "Please either set both COMMIT and SNAPSHOT_PV or neither!"
+	elif [[ -n ${SNAPSHOT_PV} && ${PV} != ${SNAPSHOT_PV} ]]; then
+		die "Rename of snapshot ebuild detected - please check COMMIT & SNAPSHOT_PV!"
+	fi
+}
+
+pkg_setup() {
+	gnome2_environment_reset # bug 543364 # TODO: read it
+}
+
+src_prepare() {
+	default
+
+	if [[ -n ${COMMIT} ]]; then
+		# This file really should be upstream's responsibility but what can you do other than hack
+		# together a work-around for an upstream's tarball generator lacking required integration?
+		if [[ -n ${SNAPSHOT_FIX_GITVERSION} && ${SNAPSHOT_FIX_GITVERSION} -ge 1 ]]; then
+			echo ${PV%_*}-${COMMIT:0:8} > .tarball-version
+		fi
+	fi
+}
+
+multilib_src_configure() {
+	local emesonargs=(
+		-Dadrian-aec=false # Not packaged?
+		--localstatedir="${EPREFIX}"/var
+		-Dmodlibexecdir="${EPREFIX}/usr/$(get_libdir)/${PN}/modules" # Was $(get_libdir)/${P}
+#		-Dsystemduserunitdir=$(systemd_get_userunitdir)
+		-Dudevrulesdir="$(get_udevdir)"/rules.d
+		-Dbashcompletiondir="$(get_bashcompdir)" # Alternatively DEPEND on app-shells/bash-completion for pkg-config to provide the value
+		$(meson_native_use_feature alsa)
+		$(meson_native_use_bool bluetooth bluez5)
+		$(meson_native_use_bool daemon)
+		$(meson_native_use_bool native-headset bluez5-native-headset)
+		$(meson_native_use_bool ofono-headset bluez5-ofono-headset)
+		$(meson_native_use_feature glib gsettings) # Supposedly correct?
+		$(meson_native_use_feature gstreamer)
+		$(meson_native_use_feature gtk)
+		$(meson_native_use_feature jack)
+		-Dsamplerate=disabled # Matches upstream
+		$(meson_native_use_feature lirc)
+		$(meson_native_use_feature orc)
+		$(meson_native_use_feature oss oss-output)
+		$(meson_native_use_feature ssl openssl)
+		# tests involve random modules, so just do them for the native # TODO: tests should run always
+		$(meson_native_use_bool test tests)
+		$(meson_native_use_feature udev)
+		$(meson_native_use_feature webrtc-aec)
+		$(meson_native_use_feature zeroconf avahi)
+		$(meson_native_use_feature equalizer fftw)
+		$(meson_native_use_feature sox soxr)
+		-Ddatabase=$(multilib_native_usex gdbm gdbm simple) # tdb is also an option but no one cares about it
+		$(meson_use forget-missing stream-restore-clear-old-devices)
+		$(meson_feature glib) # WARNING: toggling this likely changes ABI
+		$(meson_feature asyncns)
+		#$(meson_use cpu_flags_arm_neon neon-opt)
+		$(meson_feature tcpd tcpwrap) # TODO: system-wide specific?
```

Comment:
`tcpwrap` only affects daemon so this should be `meson_native_use_feature` as well

Reply:
Indeed, I wonder if this is a change since previous version or if I missed it all this time. Either way will get fixed with the next push. Thank you for noticing and reporting this.